mlocal/frags/Makefile.stub

```diff
@@ -272,9 +272,9 @@ test:
 testall: vendor-check check test
 
 .PHONY: rpm
-rpm:
+rpm: dist
 	@echo " BUILD RPM"
-	$(V)cd $(SOURCEDIR) && dist/rpm/buildrpm
+	$(V)cd $(SOURCEDIR) && rpmbuild -tb singularity-*.tar.gz
```

Comment:
Could this use a specific version number instead of '*', in case there is another tarball left from something?

Reply:
Yeah, we're currently working on getting a quick PR that's passing just the version number (i.e. 3.0.0), into the Makefile for use.

And a question for you...  This is the Travis error:

```
 DIST create tarball: singularity-3.0.0.tar.gz
*** Please tell me who you are.
Run
  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"
to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: empty ident  <testuser@travis-job-2759d569-49a5-49c8-87c0-a56463bcacef.(none)> not allowed
```

Would you know how to uhh... fix the Travis config here?

Reply:
It looks like you've gotten past that, it's now failing because it can't find go.  Unfortunately I don't have time to look at it this morning, I could this afternoon (Chicago time) if it is still not working.

Reply:
The only reason for the -V 0.0 was that some version was needed, and there wasn't any.  If you've now got a VERSION file you can remove the -V 0.0 from .travis/rpmbuild_test